src/sdk/namespace_fs.js

```diff
@@ -1426,6 +1426,15 @@ class NamespaceFS {
         }
     }
 
+    _is_force_md5_enabled(object_sdk) {
```

Comment:
I think you added this function under the "title" `// MULTIPART UPLOAD //` we also have the title `// INTERNALS //` which I think will help us in keep the file organized.